app/views/users/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>New User Form</h1>
+
+<%= form_for @user, url: {action: "create"} do |f| %>
```

Comment:
Good job with `form_for` implementation

However, you forgot to include commented out code to prove that you implemented it in the following ways:
- Regular HTML FORM
- using `form_tag` syntax

Please add back those implementations within comments. [REQUIRED]

Reply:
We left out commented code in accordance with Microverse instructions, and also you can see that code if you check the git log.